app/src/main/java/com/elmeradrianv/shesafe/fragments/ProfileUserFragment.java

```diff
@@ -0,0 +1,275 @@
+package com.elmeradrianv.shesafe.fragments;
+
+import android.content.Intent;
+import android.database.Cursor;
+import android.net.Uri;
+import android.os.Bundle;
+import android.provider.MediaStore;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.EditText;
+import android.widget.ImageButton;
+import android.widget.ImageView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.cardview.widget.CardView;
+import androidx.core.content.ContextCompat;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.load.resource.bitmap.RoundedCorners;
+import com.elmeradrianv.shesafe.LoginActivity;
+import com.elmeradrianv.shesafe.R;
+import com.elmeradrianv.shesafe.adapters.EmergencyContactsCardAdapter;
+import com.elmeradrianv.shesafe.auxiliar.MyPair;
+import com.elmeradrianv.shesafe.database.EmergencyContacts;
+import com.elmeradrianv.shesafe.database.User;
+import com.parse.ParseException;
+import com.parse.ParseFile;
+import com.parse.ParseUser;
+import com.parse.SaveCallback;
+
+import java.io.File;
+import java.util.HashMap;
+
+
+public class ProfileUserFragment extends Fragment {
+    protected EmergencyContactsCardAdapter adapter;
```

Comment:
Let's take a look at all these member variables. What ones actually need to be member variables and what ones could be local variables?